### PR TITLE
Hardcode a workaround to ignore merge commits.

### PR DESCRIPTION
**Pull request by benpatterson**
Issue 116 in the janinko fork

### CODE REVIEW

**Comment by jzoldak** [comment]
:hammer: LGTM :boat:

File: src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java [hunk 1]
```diff
@@ -53,7 +53,12 @@ public String getCommit() {
 	}
 	
 	public boolean isMerged() {
-		return merged;
+        // hardcode hack: never automatically merge a commit for a pull request build.
+        // see various bugs that are affected by this. Header descriptions for edx are in
+        // https://openedx.atlassian.net/browse/TE-703
+
+        //return merged;
+        return false;
```
Comment (jzoldak) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java:61]: Does it make more sense to instead change the object's merged attribute? I.e. where

```
this.merged = merged;
```

That way the getShortDescription function will also return the right thing?

Comment (benpatterson) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java:61]: I considered that, but wanted to leave it down here with the idea that it eventually gets fixed in this method. Plus I wanted to conserve where the attribute was coming from (the API call).

Comment (jzoldak) [inline src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java:61]: Cool, if it works then I'm good with it. :boat:

**Pull request merged**